eMPRess_GUI_version1.py

```diff
@@ -0,0 +1,354 @@
+                self.dup_cost.set(str(round(float(self.dup_cost.get()), 2)))
+                self.has_dup_cost = True
+                self.enable_recon_btn()
+        else:
+            messagebox.showinfo("Warning", "Please enter a non-negative number.")
+            self.dup_cost.set("")
+            self.has_dup_cost = False
+
+    def check_trans_input(self, *args):
+        """Checks and updates the transfer cost when user enters a value in the entry box."""
+        transValue_without_decimal_point = self.trans_cost.get().replace(".", "")
+        if str.isdigit(transValue_without_decimal_point) and ("-" not in transValue_without_decimal_point):
+                self.trans_cost.set(str(round(float(self.trans_cost.get()), 2)))
+                self.has_trans_cost = True
+                self.enable_recon_btn()
+        else:
+            messagebox.showinfo("Warning", "Please enter a non-negative number.")
+            self.trans_cost.set("")
+            self.has_trans_cost = False
+    
+    def check_loss_input(self, *args):
+        """Checks and updates the loss cost when user enters a value in the entry box."""
+        lossValue_without_decimal_point = self.loss_cost.get().replace(".", "")
+        if str.isdigit(lossValue_without_decimal_point) and ("-" not in lossValue_without_decimal_point):
+                self.loss_cost.set(str(round(float(self.loss_cost.get()), 2)))
+                self.has_loss_cost = True
+                self.enable_recon_btn()
+        else:
+            messagebox.showinfo("Warning", "Please enter a non-negative number.")
+            self.loss_cost.set("")
+            self.has_loss_cost = False
+
+    def recon_analysis(self):
+        """Displays reconciliation results in numbers and further viewing options for graphical analysis."""
+        # creates a frame for the three checkbuttons
+        recon_checkbox_frame = tk.Frame(self.func_frame)
+        recon_checkbox_frame.grid(row=3, column=0, sticky="nsew")
+        recon_checkbox_frame.pack_propagate(False)
+        self.recon_space_btn_var = tk.IntVar()
+        self.recons_btn_var = tk.IntVar()
+        self.histogram_btn_var = tk.IntVar()
+        recon_space_btn = tk.Checkbutton(recon_checkbox_frame, text="View solution space", padx=10, variable=self.recon_space_btn_var, command=self.open_and_close_window_recon_space)
+        recons_btn = tk.Checkbutton(recon_checkbox_frame, text="View reconciliations", padx=10, variable=self.recons_btn_var, command=self.open_and_close_window_recons)
+        histogram_btn = tk.Checkbutton(recon_checkbox_frame, text="Stats mode", padx=36, variable=self.histogram_btn_var, command=self.open_and_close_window_histogram)
+        recon_space_btn.pack()
+        recons_btn.pack()
+        histogram_btn.pack()
+
+        # shows reconciliation results as numbers
+        recon_nums_frame = tk.Frame(self.output_frame)
+        recon_nums_frame.grid(row=2, column=0, sticky="nsew")
```

Comment:
Comment on one of the stickies to say what it does

Reply:
Done:) (for the logo_frame when sticky first appeared)